tests/testthat/test-add_n.tbl_uvregression.R

```diff
@@ -0,0 +1,70 @@
+skip_if_not(is_pkg_installed(c("broom.helpers", "broom", "survival", "survey"), reference_pkg = "gtsummary"))
```

Comment:
I _think_ we can test the erorr messaging using a mice model. Do you want to add a test for this?